packages/mint-components/src/components/tax-and-cash/sqm-docusign-form/sqm-docusign-form.feature

```diff
@@ -0,0 +1,61 @@
+@owner:andy @author:andy
+Feature: Docusign Form
+
+  Background: A user has submitted their information in Step 1 and Step 2
+    Given a user is on the Docusign Form
+
+  @minutia
+  Scenario: Docusign URL is set in iframe on page load
+    When the Docusign Form loads
+    And the Docusign URL has been fetched
+    Then the URL set is set on the Docusign iframe
+    And the iframe loads based on the URL
+
+  @minutia
+  Scenario: Web-page fails when Docusign URL cannot be fetched
+    When the Docusign Form loads
+    And the Docusign URL cannot be fetched
+    Then the URL is not set on the Docusign iframe
+    And the iframe fails to load
+
+  @minutia @ui
+  Scenario Outline: Text copies change based on tax form participant fills out
+    When they are required to fill out <typeTaxForm>
+    Then the <taxFormSubHeader> displays with "<typeTaxForm>"
+    And the <taxFormDescriptionCopy> / <changeTaxFormCopy> changes depending on the <typeTaxForm>
+
+    Examples: 
+      | typeTaxForm | taxFormSubHeader  | taxFormDescriptionCopy                                                                                                   | changeTaxFormCopy                                                |
+      | W9          | W9 Tax Form       | Participants based in the US and partnering with US-based brands need to submit a W9 form.                               | Not based in the US?                                             |
+      | W8-BEN      | W8-BEN Tax Form   | Individuals residing outside of the US, joining the referral program of a US-based cmpany, need to submit a W8-BEN form. | Represent a business entity or you're based in the US?           |
+      | W8-BEN-E    | W8-BEN-E Tax Form | Participants residing outside of the US who represent a business enttiy need to submit a W8-BEN-E form.                  | Joining this program as an individual or you're based in the US? |
+
```

Comment:
@Locrian24 @AndiLuo we need to make sure we have a spec something that documents and makes sure the submitting of the form actually works